models/cohortdata.go

```diff
@@ -21,7 +21,7 @@ type PersonConceptAndValue struct {
 	PersonId                int64
 	ConceptId               int64
 	ConceptClassId          string
-	ConceptValueAsString    string
+	ConceptValueName        string
```

Comment:
```suggestion
	ConceptValueAsConceptName        string
```
although longer, I think this better aligns with the naming for the other variables (see the ones just below).

Reply:
I think this name is more confusing actually to me than ConceptValueName. The Observation already have a concept, but this name is the observation value, which itself is another concept. Therefore concept value('s) name

If we really need to make it more specific, maybe ObservationValueConceptName?

Reply:
Ok, I agree. I like the one you proposed, with a little extra: ObservationValue**As**ConceptName